src/pages/EntityDetails/EntityDetails.js

```diff
@@ -128,6 +133,11 @@ const EntityDetails = ({ activeView, setActiveView, type, children }) => {
                     label: "Integrations",
                     onClick: (e) => {
                       e.preventDefault();
+                      e.target.scrollIntoView({
```

Comment:
Rather than repeat this call with the same config thrice, it should be pulled out into a `handleNavClick` function.